.github/workflows/test-model.yml

```diff
@@ -0,0 +1,59 @@
+name: test-model
+
+on:
+  push:
+    branches:
+      - master
+  pull_request: null
+
+env:
+  PY_COLORS: "1"
+
+jobs:
+  tests:
+    name: tests
+    runs-on: "ubuntu-latest"
+    defaults:
+      run:
+        shell: bash -l {0}
+
+    steps:
+      - name: cancel previous runs
+        uses: styfle/cancel-workflow-action@0.6.0
```

Comment:
Why are the native concurrency settings not used here?

Reply:
I (too blindly) copied this from `tests.yml`. Will change now.

Reply:
FWIW, the GHA stuff here was done before github added that feature.